docker-compose.yml

```diff
@@ -2,12 +2,11 @@
 version: '3.7'
 services:
   pmm-managed-server:
-    image: perconalab/pmm-server:dev-container
-#    build:
-#      context: .
-#      args:
-#        PMM_VERSION: ${PMM_VERSION:-perconalab/pmm-server:dev-latest}
-#      dockerfile: devcontainer.Dockerfile
+    build:
+      context: .
+      args:
+        PMM_SERVER_IMAGE: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-container}
+      dockerfile: devcontainer.Dockerfile
```

Comment:
This doesn't have any effect on prebuilt devcontainer (since it already contains all the tools we install), but we need if $PMM_SERVER_IMAGE is set to maybe a feature build.